app/src/main/java/com/tangem/tap/features/shop/domain/models/SalesError.kt

```diff
@@ -0,0 +1,5 @@
+package com.tangem.tap.features.shop.domain.models
+
+sealed class SalesError {
+    data class GeneralSalesError(val cause: Throwable) : SalesError()
```

Comment:
Думаю стоит явно дать понять, что эта ошибка из слоя данных
```suggestion
    data class DataError(val cause: Throwable) : SalesError()
```

Reply:
ок